data/core.yaml

```diff
@@ -824,6 +827,13 @@ en:
   cannot_zoom: "Cannot zoom out further in current mode."
   full_screen: Toggle Full Screen
   QA:
+    osmose:
+      title: Osmose Issue
+      detail_title: Issue Details
+      elems_title: Issue Elements
+      fix_title: Fix Guidelines
+      trap_title: Common Mistakes
+      translation: Translations supplied by Osmose
```

Comment:
Might just say "Translate", and maybe hide the button unless there are untranslated strings onscreen.

Separately, we should add text to iD's documentation about where you can translate the Osmose strings.

Reply:
I believe there's no way for us to check whether untranslated strings are present (since the API automatically falls back to English strings and they all get cached under the current language code).

My thinking with not using a "call to action" here is that if there are issues with the strings or missing translations then a user inclined to improve them knows where they come from - and if there aren't then it's still nice to acknowledge they're supplied by a 3rd party.

Reply:
Hmm we could fetch the English strings too and see if they're equivalent? The issue for me is that the string is sort of long and overly draws my attention. "Translations by Osmose" might be a small improvement. It'd also help to put the button under the grey box or to at least give it a greater margin.

Another issue is that I _think_ we use a verb phrase everywhere else we use the outlink icon. Something like "View translations" would keep that while being less "asky". We could also use a different icon.